### PR TITLE
Adding links in menu to existing API reference for Go and Java

### PR DESCRIPTION
**Pull request by cullywakelin**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ module.exports = {
       collapsed: true,
       label: 'Go SDK',
       items: [
+        {
+          type: 'link',
+          label: 'API Reference',
```
Comment (sw-yx) [inline sidebars.js:54]: is there some way we can mark these as external links? would be a nice to have the svg icon in here, or failing which, an emoji.

**Review by sw-yx — APPROVED** [review]

File: sidebars.js (hunk 1 above)
Comment (cullywakelin) [reply sidebars.js:54]: I will take note of this and see about doing some customizations there -

Comment (sw-yx) [reply sidebars.js:54]: can use 🔗 emoji if we can't use the external link svg

**Review by rapmastersethmc — COMMENTED** [review]
Agree - we should mark external links in some way.

**Review by rapmastersethmc — APPROVED** [review]

File: sidebars.js (hunk 1 above)
Comment (cullywakelin) [reply sidebars.js]: Cool. Now looks like this:
Screen Shot 2020-11-10 at 8 49 37 AM

**Pull request merged**